webrtc/videoroom/assets/src/membraneWebRTC.ts

```diff
@@ -55,8 +55,8 @@ interface Callbacks {
     newLabel: string
   ) => void;
   onDisplayStream?: (stream: MediaStream, label: string) => void;
```

Comment:
It seems we messed something up in previous PR's. We no longer use `onReplaceStream` and `onDisplayStream`. Let's remove it